src/main/kotlin/no/skatteetaten/aurora/boober/service/openshift/ResourceMerger.kt

```diff
@@ -48,4 +51,16 @@ private fun updatePersistentVolumeClaim(mergedResource: JsonNode, existingResour
 
 private fun updateService(mergedResource: JsonNode, existingResource: JsonNode) {
     mergedResource.updateField(existingResource, "/spec", "clusterIP")
+}
+
+private fun updateNamespace(mergedResource: ObjectNode, existingResource: ObjectNode) {
+
+    val existingAnnotations = existingResource.at("/metadata/annotations") as ObjectNode
```

Comment:
Why do you do this for annotations? I though labels was the thing we were going to merge?

This looks like a very generic method to me, if we send in a parent and field it can be used for lots of things. Nice! I am not saying we should do this now, but I would not name the method updateNamespace.

Reply:
The node selector etc. is actually specified as annotations. I don't think there are other labels than the ones we create already.
We could probably do the updateNamespace method more general but the current semantics of the mergeWithExistingResource has one method pr. resource kind which is easy to read and understand. We should probably preserve that and rather extract a helper from the current implementation.